website/docs/components/accordion/partials/code/component-api.md

```diff
@@ -0,0 +1,25 @@
+## Component API
+
+### Accordion
+
+The base Accordion component serves only as a wrapper to group together one or more AccordionItem child components.
+
+### AccordionItem
+
+<Doc::ComponentApi as |C|>
```

Comment:
[blocker] the `@ariaLabel` argument is missing

Reply:
Added, thanks